microprofile/metrics-smallrye/src/main/java/org/wildfly/extension/microprofile/metrics/MicroProfileMetricsSubsystemDefinition.java

```diff
@@ -73,19 +71,27 @@ public class MicroProfileMetricsSubsystemDefinition extends PersistentResourceDe
             .setAllowExpression(true)
             .build();
 
-    static final AttributeDefinition[] ATTRIBUTES = { SECURITY_ENABLED, EXPOSED_SUBSYSTEMS, PREFIX };
+    static final AttributeDefinition[] ATTRIBUTES = {SECURITY_ENABLED, EXPOSED_SUBSYSTEMS, PREFIX};
 
     protected MicroProfileMetricsSubsystemDefinition() {
         super(new SimpleResourceDefinition.Parameters(MicroProfileMetricsExtension.SUBSYSTEM_PATH,
                 MicroProfileMetricsExtension.getResourceDescriptionResolver(MicroProfileMetricsExtension.SUBSYSTEM_NAME))
                 .setAddHandler(MicroProfileMetricsSubsystemAdd.INSTANCE)
```

Comment:
This can/should be 'new ModelOnlyAddStepHandler(ATTRIBUTES)'. MicroProfileMetricsSubsystemAdd can/should be removed.

Reply:
Done